pom.xml

```diff
@@ -1,128 +1,52 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
- <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
- <parent>
   <groupId>org.jenkins-ci.plugins</groupId>
-  <artifactId>plugin</artifactId>
-  <version>2.12</version>
-  <relativePath />
- </parent>
-
- <artifactId>foreman-node-sharing</artifactId>
- <packaging>hpi</packaging>
- <name>Foreman Node Sharing Plugin</name>
- <version>1.2.4-SNAPSHOT</version>
- <description>Attach Foreman Shared Resources as Jenkins Nodes</description>
- <url>https://wiki.jenkins-ci.org/display/JENKINS/Foreman+Node+Sharing+Plugin</url>
-
- <licenses>
-  <license>
-   <name>MIT License</name>
-   <url>http://opensource.org/licenses/MIT</url>
-  </license>
- </licenses>
-
- <developers>
-  <developer>
-   <id>scoheb</id>
-   <name>Scott Hebert</name>
-   <email>shebert@redhat.com</email>
-  </developer>
- </developers>
-
- <scm>
-  <connection>scm:git:git://github.com/jenkinsci/foreman-node-sharing-plugin.git</connection>
-  <developerConnection>scm:git:git@github.com:jenkinsci/foreman-node-sharing-plugin.git</developerConnection>
-  <url>https://github.com/jenkinsci/foreman-node-sharing-plugin</url>
-  <tag>HEAD</tag>
- </scm>
-
- <distributionManagement>
-  <repository>
-   <id>repo.jenkins-ci.org</id>
-   <url>https://repo.jenkins-ci.org/releases</url>
-  </repository>
- </distributionManagement>
-
- <repositories>
-  <repository>
-   <id>repo.jenkins-ci.org</id>
-   <url>http://repo.jenkins-ci.org/public/</url>
-  </repository>
- </repositories>
-
- <pluginRepositories>
-  <pluginRepository>
-   <id>repo.jenkins-ci.org</id>
-   <url>http://repo.jenkins-ci.org/public/</url>
-  </pluginRepository>
- </pluginRepositories>
```

Comment:
You must restore `repositories` and `pluginRepositories`. (prefer `https` while you are here)